posts/urls.py

```diff
@@ -15,6 +15,7 @@
 from .views import PostCreateView
 from .viewsfolder.feed_views import FrontEndPublicPosts
 from .viewsfolder.feed_views import FrontEndAuthorPosts
+from .viewsfolder.feed_views import FrontEndFeed
```

Comment:
https://docs.djangoproject.com/en/2.1/ref/settings/#append-slash
Add this to the setting file please.

Reply:
if you just mean putting in "APPEND_SLASH = True" into settings.py, it is done :)

Reply:
```suggestion
from .viewsfolder.feed_views import FrontEndFeedThat`
````
Sweet.

Reply:
Sweet